ptf/SAI_PTF_user-guide.md

```diff
@@ -81,6 +81,29 @@ Build SAI thrift server and libraries
 
 ```bash
 export SAITHRIFTV2=y
+export GEN_SAIRPC_OPTS="-ve" # optional - to use SAI experimental headers
+# Below is an extended example of linking additional application-specific libraries besides libsai.
+# Modify to suit a particular use-case. Omit if not needed.
+export SAIRPC_EXTRA_LIBS="\
```

Comment:
I think just one line example should be enough for user to understand the usage of this variable.

E.g:
export SAIRPC_EXTRA_LIBS="-L/usr/local/lib/ -lpthread"